keyboards/ungodly/priya50/readme.md

```diff
@@ -0,0 +1,15 @@
+# Priya50
+
+A 50% hotswap pcb powered by the RP2040, with underglow lighting, encoder and oled display support.
+
+![Priya50](https://i.imgur.com/zNnmPV9.jpg)
```

Comment:
The description does not match the photo here, nor the code. Is this a macropad or 50% keyboard? Yes or no to OLED support? You removed all the code for it.

Reply:
Sorry, wrong photos from imgur. Updated urls

Reply:
I'll remove the OLED support verbiage but it does support OLED. Messed up the I2C pins on the first iteration of the hardware but fixed in second iteration. Don't have any PCBs on hand to confirm the oled functionality so removed the code.